docs/api.md

```diff
@@ -117,8 +117,10 @@ The format of the template of one row of Pipe 2 is as follows:
   "statement": {statementBlob},
   "itemId": [itemId],
   "reference": {
-    referenceMetadata: referenceBlob,
-    extractedData: [ [value] ]
+    "url": [referenceURL],
+    "ext_id_pid": [externalIdProp],
```

Comment:
What's the logic behind pulling the `ext_id_pid` to this out object? Is it to leave it available for reuse in the matching pipes? Or for display in the game?

Reply:
Available for reuse